qa/workunits/rbd/rbd_mirror.sh

```diff
@@ -75,7 +75,7 @@ TEMPDIR=
 daemon_asok_file()
 {
     local daemon=$1
-    local cluster=$2
+    local cluster=$(daemon_local_cluster "${daemon}")
 
```

Comment:
I thing this is not right. The daemon have two asok files -- for local and remote cluster. It looks like with you change `daemon_asok_file` will generate path to local asok only?